arangosh/Import/ImportHelper.cpp

```diff
@@ -427,17 +429,27 @@ bool ImportHelper::importJson(std::string const& collectionName,
 
       // send all data before last '\n'
       char const* first = _outputBuffer.c_str();
-      char* pos = (char*)memrchr(first, '\n', _outputBuffer.length());
+      char const * pos = (char*)memrchr(first, '\n', _outputBuffer.length());
```

Comment:
```suggestion
      char const* pos = static_cast<char const*>(memrchr(first, '\n', _outputBuffer.length()));
```

Reply:
done

Reply:
mmh.... seems *not* done, according to Github.